### ISSUE TITLE
How do you plant seeds?

### ISSUE THREAD

**Issue by MusicallyInspired** [jdah/microcraft#15]
Using a hoe on grass just digs up the ground like a shovel. Not sure how to cultivate it to plant seeds.

**Comment by jdah** [comment]
If you "hit" the cultivated ground with the seeds you plant them :)

**Comment by MusicallyInspired** [comment]
Yeah but I can't cultivate the ground. It digs the ground instead. So "hitting" it with seeds does nothing.

**Comment by jdah** [comment]
Oh that's strange, I'll look into this.

**Comment by jdah** [comment]
I just reread the code, you apparently have to hit it with the hoe twice to make it into farmland :) what a silly system I made. Does that work?

**Comment by MusicallyInspired** [comment]
Oh! Pft. Silly me for not trying that. I thought I had. Yes, hoe-ing the ground a second time does indeed cultivate. Sorry! :)
